scheduler/src/cook/rest/api.clj

```diff
@@ -1896,8 +1896,10 @@
      [ctx]
      ;; the client is expected to cache the redirect location
      (if-let [leader-url (::leader-url ctx)]
-       (let [request-path (get-in ctx [:request :uri])]
-         [true {:location (str leader-url request-path)}])
+       (let [request-path (get-in ctx [:request :uri])
+             request-param (get-in ctx [:request :query-string])]
```

Comment:
Maybe `query-string` instead of `request-param`?

Reply:
Changed.